### DISCUSSION TITLE
YoutubeShortcode width

### DISCUSSION THREAD

**Discussion by crapitan** [statiqdev/Discussions#41] [Q&A] [answered]
Hi, 

How do I set the width of an embedded youtube video?

**Comment by daveaglick** [comment, marked as answer]
Short answer: right now you can't 😦.

Longer answer: I'm going to assume you mean when using the YouTube shortcode here since that's the only build-in Statiq embedding support. The YouTube embed is generated using a standard called [oEmbed](https://oembed.com/) that lets a variety of providers (including YouTube) "tell" a site what code to embed for content from that site. The oEmbed specification does provide a `maxwidth` and `maxheight` parameter that the caller can supply. I've added an issue to support those in the next release of Statiq: https://github.com/statiqdev/Statiq.Web/issues/943. So while you can't specify this today, you'll be able to soon.

**Reply by crapitan** [reply]
Great thanx!
